index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Abel mobile setup </title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+      <div class="home-page">
+        <header>
+          <nav id="nav">
+            <a class="logo" href="#">Abel</a> 
+
+            <nav class="hamburger"><img src="image/Union.png" alt="Menu"></nav>
+            <ul class="internal-link">
+               <li><a href="#my_work">Portfolio</a></li>
+               <li><a href="#about_me">About</a></li>
+               <li><a href="#form-section"> Contact </a></li>
+            </ul>
+
+          </nav>
+        </header>
+        <section id="home_presentation">
+           <h4 class="short-presentation">I’m Abel Fikre
```

Comment:
- [ ] Please use the `h1` tag for the main heading at once on one page so kindly replace `h4` with `h1` tag because Google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.